README.md

```diff
@@ -32,6 +32,17 @@ cd $GOPATH/src/github.com/terraform-providers/terraform-provider-gcore
 make build
 ```
 
+Building the provider locally
```

Comment:
why need this one, I believe it's confusing, does not the current guide work ?

Reply:
removed